phy/cluster/default_settings.py

```diff
@@ -7,47 +7,48 @@
 # Spike detection
 # -----------------------------------------------------------------------------
 
-def spikedetekt_params(sample_rate):
-
-    assert sample_rate > 0
+traces = {
+    'raw_data_files': [],
+    'n_channels': None,
+    'dtype': None,
+    'sample_rate': None,
+}
 
-    return dict(
-        # Filter.
-        filter_low=500.,
-        filter_high=0.95 * .5 * sample_rate,
-        filter_butter_order=3,
 
-        # Data chunks.
-        chunk_size=int(1. * sample_rate),
-        chunk_overlap=int(.015 * sample_rate),
+spikedetekt = {
+    'filter_low': 500.,
+    'filter_high_factor': 0.95 * .5,  # will be multiplied by the sample rate
```

Comment:
> No. In fact, I don’t think this is necessary at all. It has always been there, since Jozsef Csicsvari’s original code, but I have never seen the point in using a bandpass rather than simply a highpass filter.

OK -- considering that for the next release...